src/canopy-webpack-config.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-env node */
+const path = require('path');
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+const merge = require('webpack-merge')
+
+module.exports = function(name, overridesConfig) {
+  if (typeof name !== 'string') {
+    throw new Error('canopy-webpack-config expects a string name as the first argument')
+  }
+
+  if (typeof overridesConfig !== 'object') {
+    throw new Error('canopy-webpack-config expects an object as a second argument to override the canopy defaults. Received ' + typeof overridesConfig)
+  }
+
+  const defaultCanopyConfig = {
+    entry: `./src/${name}.js`,
+    output: {
+      filename: `${name}.js`,
+      library: name,
+      libraryTarget: 'amd',
+      path: path.resolve(process.cwd(), 'build'),
```

Comment:
I switched from having a "sofe" directory to having a "build" directory.

The reason is that, talking with a few people, I realized that people were understandably assuming that sofe was somehow involved in our webpack build, which is not true.

What this means is that people who use this will have to change their gitlab-ci.yml files to reference the build directory instead of sofe. See https://code.canopy.ninja/front-end/blue/login-ui/merge_requests/103/diffs#587d266bb27a4dc3022bbed44dfa19849df3044c_46_45 as an example.

Reply:
Won't you also need to add `build` to the `.gitignore` files?

Reply:
yeah